src/modules/70-pipeline/context/StageFormContext.tsx

```diff
@@ -12,6 +12,11 @@ import type { PipelineInfoConfig } from 'services/pipeline-ng'
 export interface StageFormContextValues {
   getStageFormTemplate<T>(path: string): T | PipelineInfoConfig
   updateStageFormTemplate<T = unknown>(data: T, path: string): void
+  pipelineGitMetaData?: {
```

Comment:
```
type PipelineGitMetaData = {
  branch: string
  repoName: string
  connectorRef: string
}
```
```suggestion
  pipelineGitMetaData?: PipelineGitMetaData
```
And the use PipelineGitMetaData at all other places where these values need to be passed

Reply:
Done